packages/font/package.json

```diff
@@ -25,6 +25,6 @@
     "fontkit": "2.0.2"
   },
   "peerDependencies": {
-    "next": "*"
+    "next": "14.3.0-canary.15"
```

Comment:
This was causing another version of Next.js to be installed, i.e. 14.2.1 and as such caused conflicts and sometimes resolved to the older version incorrectly.